dashboard/api_test.py

```diff
@@ -799,3 +803,461 @@ def test_program_no_courses(self, mock_info_course):
             "courses": []
         }
         self.assertEqual(res, expected_data)
+
+
+class CertificatesTests(ESTestCase):
```

Comment:
again, the tests look similar but some small part are pretty different, so move them in a single class can result in more work than this.